packages/svelte2tsx/svelte-shims.d.ts

```diff
@@ -94,3 +97,14 @@ declare function __sveltets_each<T>(
     array: ArrayLike<T>,
     callbackfn: (value: T, index: number) => any
 ): any;
+
+declare class Svelte2TsxComponent<Props = {}, Slots = {}, Events = {}> {
```

Comment:
it would be ideal for this to also extend SvelteComponent, maybe this can be an interface instead? 
We want instanceOf checks to work instead of returning ("This can never be true"), it would also make our exported .d.ts files generated from this compatible with the svelte component api 
(I know the old code didn't do this, but that was due to type conflicts I was having with svelte @types, I think this was resolved)

Reply:
Yeah was thinking the same, we should extend from the base class

Reply:
Mhm, we cannot extend from it, because we need to redeclare the component ourselves at the top. But we can at least copy the interface and constructor.